src/code-a-long/calculator/calc-grid.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
```

Comment:
update title name to describe what the project is

Reply:
Thanks, will do

Reply:
Ok, I added my NASA page. I'm not sure if I need to make another pull request, but let me know if I do